src/features/landing/components/Footer/CopyrightNoticeAndLinks/CopyrightNoticeAndLinks.tsx

```diff
@@ -0,0 +1,32 @@
+'use client';
+
+import { Box, Stack, Typography } from '@mui/material';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import NavList from '../NavList/NavList';
+
+function CopyrightNoticeAndLinks() {
```

Comment:
There is no need to move to another component, need to return this code to the footer